### PR TITLE
first quick shot to improve the windows install 

### PR DESCRIPTION
**Pull request by jensens**
see #144, on windows:

- Do not install instance scripts.
- Generate a working wsgi.ini ready for runwsgi
- add a note to the docs

- at the moment it does not does not respect ip/port (todo)

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: README.rst [hunk 1]
```diff
@@ -668,6 +668,22 @@ http-header-max-length
...
 ============================
 
+Windows
+-------
+
+On the windows platform the ``bin/instance`` script as described below will not be generated, because it uses a Unix specific implementation.
+
+To run Plone start it with::
+
+  .\bin\runwsigi.exe -v .\parts\etc\wsgi.ini
```
Comment (mauritsvanrees) [inline README.rst:678]: Typo: `runwsigi` -> `runwsgi`. Could be fixed after merge.

File: README.rst [hunk 2]
```diff
@@ -668,6 +668,22 @@ http-header-max-length
...
+
+Or for development in debug mode use
+
+  .\bin\runwsigi.exe -v .\parts\etc\wsgi.ini
```
Comment (mauritsvanrees) [inline README.rst:682]: Typo: `runwsigi` -> `runwsgi`. Could be fixed after merge.

**Comment by mauritsvanrees** [comment]
I have fixed the typos after merge and have released 6.7.5.

**Pull request merged**